core/src/main/java/jenkins/security/LastGrantedAuthoritiesProperty.java

```diff
@@ -148,10 +148,6 @@ protected void loggedOut(@Nonnull String username) {
 
     @Extension
     public static final class DescriptorImpl extends UserPropertyDescriptor {
-        public String getDisplayName() {
-            return null;    // not visible
```

Comment:
That rely on being able to return null to hide from display? Or that rely on checking `someoneElsesDescriptor.getDisplayName() == null`? Do you have any references offhand? I looked for UI code that treated descriptors with a null display name specially and did not find any, though it is hard to know exactly what to search for.